backend/fms_core/template_importer/row_handlers/sample_submission/pool.py

```diff
@@ -15,6 +15,11 @@ def process_row_inner(self, samples_info, pool, reception_date, comment):
                                             f"Make sure SampleSubmission sheet Pool Name column values "
                                             f"match a value in PoolSubmission sheet Pool Name column.")
         else:
+            # Validate that all libraries have the same platform
+            set_platform = set(sample['library'].platform_id for sample in samples_info)
+            if len(set_platform) > 1:  # len(set_kind) > 1 => not all same platform
+                self.errors["source_sample"] = (f"Source samples in pool {pool['name']} must be have the same platform .")
```

Comment:
Since the pool is already made i would say "Libraries in pool ..." and there is an extra "be" and an extra space before the ".".